Makefile

```diff
@@ -92,3 +95,47 @@ delete-awsfederatedaccountaccess: check-aws-account-id-env
 
 .PHONY: test-awsfederatedaccountaccess
 test-awsfederatedaccountaccess: check-aws-account-id-env create-account create-awsfederatedrole create-awsfederatedaccountaccess test-switch-role delete-awsfederatedaccountaccess delete-awsfederatedrole delete-account
+
+# Create CCS (BYOC) namespace
+.PHONY: create-ccs-namespace
+create-ccs-namespace:
+	# Create namespace
+	@oc process -p NAME=${CCS_NAMESPACE_NAME} -f hack/templates/namespace.tmpl | oc apply -f -
+
+# Delete CCS (BYOC) namespace
+.PHONY: delete-ccs-namespace
+delete-ccs-namespace:
+	# Create namespace
+	@oc process -p NAME=${CCS_NAMESPACE_NAME} -f hack/templates/namespace.tmpl | oc delete -f -
+
+# Create CCS (BYOC) Secret
+.PHONY: create-ccs-secret
+create-ccs-secret:
+	# Create CCS Secret
+	./hack/scripts/aws/rotate_iam_access_keys.sh -p osd-staging-2 -u osdCcsAdmin -a ${OSD_STAGING_2_AWS_ACCOUNT_ID} -o ccs-secret.yaml
```

Comment:
`ccs-secret.yaml` ends up being created in the root of the code base, and stays there if the make target fails before getting to the `delete-ccs-secret` target for whatever reason.  This could accidentally lead to credentials committed to the repository if someone is not expecting the file to be there, or not paying attention.

Maybe the creds could be written from STDIN to avoid a file being written?  Or they could be placed in /tmp, so they're cleaned up on reboot if there's a deletion failure?  Or, better, private tmp, on the (slim) chance that the user is on a shared machine.

Reply:
Good catch, I did think that these should go to stdout I will do that its the right thing to do.